README.md

```diff
@@ -1,24 +1,72 @@
-# README
+# Budgeta
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+A simple application developed in Ruby on Rails that helps you keep track of your expenses.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Rails
+- PostgresSQL
+- Tailwind CSS
 
-* System dependencies
+## Getting Started
 
-* Configuration
+To get a local copy up and running follow the steps below.
 
-* Database creation
+### Prerequisites
+- Ruby
+- Rails
 
-* Database initialization
+### Setup
+- `git clone` this repo
+- `cd` into the generated directory
+- Run `bundle install`
+- Run `rails db:create` 
+- Run `rails db:migrate` 
+- Run `rails s` 
+Note: In case this doesn't work, try `rails db:drop` and do the steps all over again
 
-* How to run the test suite
+### Install
+- Run `bundle install` to install all the necessary dependencies
+  
+### Tests
 
-* Services (job queues, cache servers, search engines, etc.)
+- Open a terminal and cd into the project folder.
+- Run `rspec ./spec`
 
-* Deployment instructions
+### Usage
+- Run `rails s` to expose a local webserver
 
-* ...
+### Demo
+- [Video Demo](https://www.loom.com/share/8c12fa3bafc34cc28052cafccaa495a3)
+- [Link Demo](https://rtonata-budgeta.herokuapp.com/)
+
+## Authors
+
+👤 **Richard Chambula**
+
+- GitHub: [@rtonata88](https://github.com/rtonata88)
+- Twitter: [@rtonata](https://twitter.com/rtonata)
+- LinkedIn: [Richard Chambula](https://www.linkedin.com/in/richard-chambula-49198425/)
+
+## License and Attributions
+
+This work is licensed under a [Creative Commons Attribution-NonCommercial 4.0 International License](http://creativecommons.org/licenses/by-nc/4.0/)
+
+## Acknowledgements
+
+Original design idea by [Gregoire Vella on Behance.](https://www.behance.net/gregoirevella)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/oliverscz/blog-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] You have added the MIT license link in the readme but there is no MIT.md file itself. So please add the license file also 👍